src/components/FolderCreationModal.jsx

```diff
@@ -26,11 +27,10 @@ const FolderCreationModal = ({
   pagesData,
   siteName,
   setIsFolderCreationActive,
-  setRedirectToNewPage,
-  setNewPageUrl,
 }) => {
   // Errors
   const [errors, setErrors] = useState('')
+  const { setShouldRedirect, setRedirectUrl } = useRedirectHook()
```

Comment:
unused `useRedirectHook` - is this here to anticipate future use?

Reply:
Yeah, will add in when save functionality is added